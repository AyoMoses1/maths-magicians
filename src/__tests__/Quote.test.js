import { render, screen } from "@testing-library/react";
import Quote from './../routes/Quote';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});


test('It should quote', () => {
  render(<Quote/>)
  const textElement = screen.getByText("Mathematics is not about numbers, equations, computations, or algorithms: It is about understanding. -William Paul Thurston")
  expect(textElement).toBeInTheDocument
})
