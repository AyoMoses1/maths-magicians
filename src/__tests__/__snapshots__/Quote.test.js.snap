// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quote-wrapper"
>
  Mathematics is not about numbers, equations, computations, or algorithms: It is about understanding. -William Paul Thurston
</div>
`;
